### ISSUE TITLE
Not able to see schema after doing migration

### ISSUE THREAD

**Issue by Saurabh-704** [cybertec-postgresql/ora_migrator#29]
Hey @laurenz .
I am running -
SELECT db_migrate(
   plugin => 'ora_migrator',
   server => 'oradb2',
   only_schemas => '{DSI}'
);

And after running it is saying successfully run.
Output -
NOTICE:  Creating staging schemas "fdw_stage" and "pgsql_stage" ...
NOTICE:  Creating foreign metadata views in schema "fdw_stage" ...
NOTICE:  Creating schemas ...
NOTICE:  Creating sequences ...
NOTICE:  Creating foreign tables ...
NOTICE:  Creating UNIQUE and PRIMARY KEY constraints ...
NOTICE:  Creating FOREIGN KEY constraints ...
NOTICE:  Creating CHECK constraints ...
NOTICE:  Creating indexes ...
NOTICE:  Setting column default values ...
NOTICE:  Dropping staging schemas ...
NOTICE:  Migration completed with 0 errors.

Successfully run. Total query runtime: 4 min 46 secs.
1 rows affected.

But I am not able to see any DSI schema under Schema section.

Before this I ran same querry for TEST schema, here I am able to get the table and table data but I am not able to see the functions which I created.

Can you please help me with this!!?

**Comment by laurenz** [comment]
The query run to find the Oracle schemas is

    SELECT username FROM dba_users;

System schemas will be excluded, but `DSI` is not among them.

What do you get for

   SELECT * FROM fdw_stage.schemas;

**Comment by Saurabh-704** [comment]
Hello @laurenz , Thank you for your response.

Let me attach the screenshot-
<img width="329" alt="o1" src="https://user-images.githubusercontent.com/47383623/90239141-b2600100-de44-11ea-8d0f-e331c7f280b3.png">

**Comment by laurenz** [comment]
Yes, that gets deleted by `db_migrate`.

Run only the `db_migrate_prepare` step, that will create the staging schemas.

**Comment by Saurabh-704** [comment]
Hey @laurenz , I am bit confused like here. Do I need to run this -

SELECT db_migrate_prepare(
   plugin => 'ora_migrator',
   server => 'oradb2',
   staging_schema => 'TEST',
   pgstage_schema => 'XYZ'
   only_schemas => '{DSI}'
);

If yes!, then here what is -
staging_schema  and pgstage_schema?!

Like I have gone through this link - https://github.com/cybertec-postgresql/db_migrator#detailed-description-of-the-migration-functions
but didnt got the exact purpose of that two things

**Comment by laurenz** [comment]
Looks good, except I would leave the default values for `staging_schema` and `pgstage_schema`.

With the values you suggest, the foreign table `schemas` would be in schema `"TEST"` in the PostgreSQL database.

**Comment by Saurabh-704** [comment]
Ok, Thank you @laurenz for your time. It means a lot.

**Comment by TimotheeWright** [comment]
Hello @laurenz,

I don't know why Saurabh-704 didn't post the solution to this problem before closing but I'm now facing the same issue.

I already manage to use ora_migrator and I wrote down every step I made to make it work. I'm currently trying to migrate another project and followed all the necessary steps and ended up with

`reveilfdw=# SELECT db_migrate(plugin => 'ora_migrator', server => 'oraclereveil',  only_schemas => '{reveil}');
NOTICE:  Creating staging schemas "fdw_stage" and "pgsql_stage" ...
NOTICE:  Creating foreign metadata views in schema "fdw_stage" ...
NOTICE:  Creating schemas ...
NOTICE:  Creating sequences ...
NOTICE:  Creating foreign tables ...
NOTICE:  Creating UNIQUE and PRIMARY KEY constraints ...
NOTICE:  Creating FOREIGN KEY constraints ...
NOTICE:  Creating CHECK constraints ...
NOTICE:  Creating indexes ...
NOTICE:  Setting column default values ...
NOTICE:  Dropping staging schemas ...
NOTICE:  Migration completed with 0 errors.
 db_migrate
------------
          0
(1 ligne)`

But nothing is created unlike the first time I migrated my project. 

I'm kind of stuck, any help would be appreciated.
